docs/index.md

```diff
@@ -1,131 +1,250 @@
-
-```toml title="pixi.toml" hl_lines="6-8"
---8<-- "docs/source_files/pixi_workspaces/introduction/deps_add/pixi.toml"
-```
-
-We can now create a Python script which uses the `cowpy` library.
-
-```py title="hello.py"
---8<-- "docs/source_files/pixi_workspaces/introduction/deps_add/hello.py"
-```
-
-The dependencies are installed in a Pixi environment.
-In order to run a command within an environment, we prefix it with `pixi run`.
-
-```bash
-pixi run python hello.py
-```
-
+Install your favorite tools with a single command.
+```shell
+pixi global install git gh nvim ipython btop bat starship ripgrep
 ```
- __________________
-< Hello Pixi fans! >
- ------------------
-     \   ^__^
-      \  (oo)\_______
-         (__)\       )\/\
-           ||----w |
-           ||     ||
+![Pixi Global Tools demo](assets/pixi_global_demo.gif)
 
-```
+---
 
 
-You can also put this run command in a **task**.
+## ↔️ What is the difference with Pixi?
 
-```bash
-pixi task add start python hello.py
-```
+| Builtin Core Features | Pixi | Conda | Pip | Poetry | uv |
+|-----------------------|---|---|---|---|---|
+| Installs Python | ✅ | ✅ | ❌ | ❌ | ✅ |
+| Supports more than Python | [✅]("Using the conda ecosystem Pixi installs any type of package, not just Python!") | ✅ | [❌]("Only Python packages") | [❌]("Only Python packages") |[❌]("Only Python packages") |
+| Cross-platform Task Runner | [✅](workspace/advanced_tasks.md "Run shell commands on all platforms with `pixi run`") | ❌ | ❌ | ❌ | ✅ |
+| Lockfiles | [✅](workspace/lockfile.md) | ❌ | ❌ | ✅ | ✅ |
+| Project Management | [✅](reference/pixi_manifest.md) | ❌ | ❌ | ✅ | ✅ |
 
-```toml title="pixi.toml" hl_lines="6-7"
---8<-- "docs/source_files/pixi_workspaces/introduction/task_add/pixi.toml"
-```
+---
 
-After adding the task, you can run the task using its name.
-
-```bash
-pixi run start
-```
 
-```
- __________________
-< Hello Pixi fans! >
- ------------------
-     \   ^__^
-      \  (oo)\_______
-         (__)\       )\/\
-           ||----w |
-           ||     ||
+## 🛠️ Installation
 
-```
+To install `pixi`, run:
 
+=== "Linux & macOS"
+    ```bash
+    curl -fsSL https://pixi.sh/install.sh | sh
+    ```
 
-You now know how to add dependencies and tasks to your environment.
-Put the workspace folder on a different machine, and you will find that Pixi will be able to fully reproduce your setup.
+=== "Windows"
+    ```powershell
+    powershell -ExecutionPolicy ByPass -c "irm -useb https://pixi.sh/install.ps1 | iex"
+    ```
 
-If you want to learn more about Pixi, check out the next page!
+!!! tip "Now restart your terminal or shell!"
+    The installation needs to become effective by restarting your terminal or sourcing your shell.
+
+??? question "Don't trust our link? Check the script!"
+    You can check the installation `sh` script: [download](https://pixi.sh/install.sh) and the `ps1`: [download](https://pixi.sh/install.ps1).
```

Comment:
Often got this question and saw that other tools share a link to view it so that's why I added this.

Reply:
technically, you could still serve arbitrary scripts on your site.
how about adding the corresponding headers on `https://pixi.sh/install.sh` s.t. it isn't downloaded automatically by your browser but instead you can view it in raw?

still, adding a link to the source on GitHub is nice to have

Reply:
when requesting https://raw.githubusercontent.com/prefix-dev/pixi/refs/heads/main/.taplo.toml, i get the headers `Content-Type: text/plain; charset=utf-8`

<img width="652" alt="image" src="https://github.com/user-attachments/assets/515b2a1a-d602-4913-b67b-f57991dfdc37" />

comparing it to https://pixi.sh/install.sh: 
<img width="421" alt="image" src="https://github.com/user-attachments/assets/3e3fea41-f4c9-4bcd-a5c0-deb1b3bb5327" />

it's probably not possible to serve these headers in github pages, though... you might be able to redirect to a self-hosted file somewhere?

Reply:
I really want to solve that, but I'm first going to focus on this PR as I don't see an easy to do this from within the GitHub pages ;)